### DISCUSSION TITLE
Questions about wiki.js backups and 'dumps'

### DISCUSSION THREAD

**Discussion by bkrepp-ctps** [requarks/wiki#5957] [Help / Questions] [answered]
I'm a bit confused about how to properly configure backups and 'dumps' of my wiki on wiki.js, and have some specific questions.
First, here are the details of the system my wiki.js is running on:

- Digital Ocean droplet
- OS - Ubuntu Linux version 20.02
- WIki.js version - 2.5.92
- Database - PostgreSQL version 11.16

Under Admin->Storage, I've activated a Local File System target as follows:
- Path: /tmp/wiki_dump
- 'Create daily backups' - enabled
Under 'Sync direction: 'Push to target"

Now, my questions:

1. Under 'Actions', if I click 'Dump all content to disk' - **Please clarify where the dumped content is being stored?** Should I expect to find it under /tmp/wiki_dump, or elsewhere? If elsewhere, where?
2. Under 'Actions', if I click 'Create Backup', the text on the page says that the manual backup will be created in a "subfolder named _manual", but again it's not 100% clear where this folder will be - Is it a subfolder of my /tmp/wiki_dump folder or someplace else? If the later, where?

Thanks for any help you can offer with these questions.
I've found wiki.js to be a great platform so far, but this part of the documentation is a little bit less than completely clear.
All best,
Ben

**Comment by fnafox** [comment, marked as answer]
Everything is relative to the configured path.
- 'Dump all content to disk' exports all of your wiki content (pages and assets) as files into configured path
- 'Create Backup' creates a backup from the exported content into a subdirectory `_manual` of the configured path

**Reply by shirgho** [reply]
Sorry for pickup up and old post and message.

But should this be done in addition to a database dump and restore?
